packages/blocks/src/page-block/default/utils.ts

```diff
@@ -178,8 +181,30 @@ function binarySearchBlockEditingState(
         assertExists(blockRect);
 
         if (!options?.skipX) {
-          if (x < detectRect.left || x > detectRect.left + detectRect.width) {
-            return null;
+          if (dragging) {
+            if (block.depth) {
+              let depth = Math.floor((blockRect.left - x) / 26);
```

Comment:
what is 26?

Reply:
padding-left

Reply:
I think we should put everything related into a constant